modules/test/playwright/tests/login-web/update_password.spec.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {captchaConfigPageTest} from '../../fixtures/captchaConfigPageTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import {clickAndExpectToBeVisible} from '../../utils/clickAndExpectToBeVisible';
+import performLogin, {performLogout} from '../../utils/performLogin';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	captchaConfigPageTest,
+	loginTest()
+);
+
+test.afterEach(
+	'Reset configuration and delete emails',
+	async ({captchaConfigPage}) => {
+		await captchaConfigPage.resetCaptchaConfiguration();
+	}
+);
+
+test('LPD-52234: Check if you can change languages in the update password page', async ({
+	apiHelpers,
+	captchaConfigPage,
+	page,
+}) => {
+	const userAccount = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	try {
+		await captchaConfigPage.disableCaptcha();
+
+		await performLogout(page);
+
+		await page.goto(liferayConfig.environment.baseUrl);
+		await page.getByRole('button', {name: 'Sign In'}).click();
+		await page.getByText('Forgot Password').click();
+		await page.waitForTimeout(1000);
+		await page.getByLabel('Email Address').fill(userAccount.emailAddress);
+		await page.getByRole('button', {name: 'Send New Password'}).click();
+
+		await performLogin(page, 'test');
+
+		const ticket =
+			await apiHelpers.headlessAdminUser.getUserAccontPasswordResetTicket(
+				userAccount.id
+			);
+
+		await performLogout(page);
+
+		await page.goto(
+			`${liferayConfig.environment.baseUrl}/c/portal/update_password?ticketId=${ticket.id}&ticketKey=${ticket.key}`
+		);
+
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: page.getByRole('menuitem', {name: 'português-Brasil'}),
+			trigger: page.getByTitle('Select a Language', {exact: true}),
+		});
+
+		expect(
```

Comment:
This `expect` instantly passes before the page can load, so it's not validating anything.  After the `clickAndExpectToBeVisible`, we should `await` until an expected element is present, like the "Change Password" heading, but probably in Brazilian Portuguese.